src/handler-builder.ts

```diff
@@ -35,6 +35,13 @@ import * as remoteConfig from './providers/remoteConfig';
 import * as storage from './providers/storage';
 import * as testLab from './providers/testLab';
 
+/**
+ * The `HandlerBuilder` class facilitates the writing of functions by producers
+ * of Firebase Extensions (currenly in limited beta).
+ * 
+ * **Do not use `HandlerBuilder` when writing normal functions for deployment via
+ * Cloud Functions for Firebase.** For normal purposes, use `FunctionBuilder`.
+ */
 export class HandlerBuilder {
   constructor() {}
```

Comment:
Can you add 
`/** @hidden */`
above the constructor? 

It should not show up in the docs.